src/components/grid-aware/DonationBlock/DonationBlock.stories.js

```diff
@@ -8,25 +9,29 @@ export default {
 
 const Template = ({
   mainTitle,
+  givingTuesdayImage,
   mainDescription,
+  givingTuesdayParagraph,
   whyDonateTitle,
   whyDonateList,
   impactTitle,
   impactList,
 }) => (
   <DonationBlock
     mainTitle={mainTitle}
+    givingTuesdayImage={givingTuesdayImage}
     mainDescription={mainDescription}
+    givingTuesdayParagraph={givingTuesdayParagraph}
     whyDonateTitle={whyDonateTitle}
     whyDonateList={whyDonateList}
     impactTitle={impactTitle}
     impactList={impactList}
   />
 );
 
-export const Default = Template.bind({});
+export const DonationBlockDefault = Template.bind({});
```

Comment:
Is there a reason you renamed this?

Reply:
Other storybook components do not just have "Default" but its component name appended with "Default" at the end.